resolwe/permissions/mixins.py

```diff
@@ -47,9 +47,12 @@ def get_fields(serializer_self):  # pylint: disable=no-self-argument
 
             def to_representation(serializer_self, instance):  # pylint: disable=no-self-argument
                 """Object serializer."""
-                # TODO: These permissions queries may be expensive. Should we limit or optimize this?
                 data = super().to_representation(instance)
-                data['current_user_permissions'] = get_object_perms(instance, self.request.user)
+
+                if 'fields' not in self.request.query_params \
```

Comment:
Use brackets instead of backslash.

From PEP8:
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.